packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -80,6 +81,17 @@ export interface IPluginLedgerConnectorBesuOptions
   logLevel?: LogLevelDesc;
 }
 
+// export interface Log{
```

Comment:
@awadhana I commented on the other PR accidentally but it was meant to be done here: please remove the commented interface definition here (no reason to keep it around)